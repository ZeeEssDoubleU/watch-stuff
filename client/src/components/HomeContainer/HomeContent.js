import React, { Component } from "react";

import './HomeContent.scss';
import VideoGrid from "../VideoGrid/VideoGrid";
import SideBarContainer from '../SideBarContainer/SideBarContainer';

class HomeContent extends Component {
	render() {
		return (
				<div className="home-content">
					<div className="responsive-video-grid-container">
						<VideoGrid title="Trending" />
						<VideoGrid title="Autos & Vehicles" hideDivider={true} />
					</div>
				</div>
		);
	}
}

export default HomeContent;
