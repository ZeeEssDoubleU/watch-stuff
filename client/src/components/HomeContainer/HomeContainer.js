import React, { Component } from "react";

import './HomeContainer.scss';
import HomeContent from "./HomeContent";
import SideBarContainer from '../SideBarContainer/SideBarContainer';

class HomeContainer extends Component {
	render() {
		return (
			<>
				<SideBarContainer />
				<HomeContent />
			</>
		);
	}
}

export default HomeContainer;
