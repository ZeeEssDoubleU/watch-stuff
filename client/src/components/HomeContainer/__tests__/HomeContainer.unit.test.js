import React from "react";
import { shallow } from "enzyme";

import HomeContainer from "../HomeContainer";

describe('HomeContainer', () => {
   test("renders", () => {
      const wrapper = shallow(<HomeContainer />);
      expect(wrapper).toMatchSnapshot();
   });
})

