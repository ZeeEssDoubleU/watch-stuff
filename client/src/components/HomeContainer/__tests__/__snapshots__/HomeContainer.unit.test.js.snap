// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer renders 1`] = `
<Fragment>
  <SideBarContainer />
  <HomeContent />
</Fragment>
`;

exports[`renders HomeContainer 1`] = `
<Fragment>
  <SideBarContainer />
  <HomeContent />
</Fragment>
`;
